README.beginner-developer.md

```diff
@@ -0,0 +1,76 @@
+# Tlf ham radio contest logger
+
+## Summary
+
+This is some collection to help recently joined coders to start with contributing TLF.
+You can try the hard way. Then you just do not need to read any further.
+Beleive me, at least for me, added unnecessary frustration not knowing any such collection.
+You lean back, feeling (almost) on top of the World -- just to have to
+learn somewhat later when asking for merge your work, that you are far from done...
+
+In spite of every effort, this doc will (and can) never become complete.
+Any comments, critisisms, suggestions, improvement requests are _ W E L C O M E .
+
+To begin with, each project using any computer language, ususally is built upon
+* partly some project conventions
+* partly some personal tastes.
+
+Individual tastes are no problem for one-man projects.
+However, for bigger and long-lasting projects, individual tastes obviously
+do not help readibility or maintenability.
+My personal note, project conventions are also mainly built up of
+personal taste of somebody in the project much before you joined ;-) ...
+
+The following is the list we could collect so far.
+This list is only numbered for easy reference.
+This list is not considered to be sorted upon importance,
+nor do I think there could be any such importance.
+Please never re-number this list, that is, only add new points at end of file.
+In case anybody feels in absolute need of picking some points upon importance,
+please simply use a list in the header immediately below this introduction, like:
+
+**`important by HA5SE: 	1, 2, 3		// sample`**
+
+
+
+## Currently documented conventions
+
+
+1)  TLF is a combination of old code mixed with many new updates.
+    Old code is as-is, you should never count on conventions in old
+    code would still continue to be considered as current.
+
+    Recent code often uses different conventions.
+
+    Code is considered old when before ...
```

Comment:
would be very thankful if somebody would please insert a year number here...

Reply:
Re travis: it is used for continuous integration, the expression in the MD file shows the latest build status. 
Their statement is that open source projects can stay, so no immediate action is needed in my opinion. Until now all worked as before.

Reply:
Just FYI: Github has an [own CI](https://docs.github.com/en/actions/guides/setting-up-continuous-integration-using-workflow-templates), we can leave Travis.